packages/fields-wysiwyg-tinymce/src/views/Field.js

```diff
@@ -1,35 +1,70 @@
 /** @jsx jsx */
 
-import { jsx } from '@emotion/core';
-import { Component } from 'react';
+import { Global, css, jsx } from '@emotion/core';
 
 import { FieldContainer, FieldLabel } from '@arch-ui/fields';
 
-import Editor from './Editor';
+// MUST IMPORT for TinyMCE to work!
+// eslint-disable-next-line no-unused-vars
+import tinymce from 'tinymce/tinymce';
 
-export default class WysiwygField extends Component {
-  onChange = value => {
-    if (typeof value === 'string' && value !== this.props.value) {
-      this.props.onChange(value);
+import { Editor } from '@tinymce/tinymce-react';
+
+const defaultOptions = {
+  autoresize_bottom_margin: 20,
+  base_url: '/tinymce-assets',
+  branding: false,
+  menubar: false,
+  plugins: 'link lists code autoresize paste quickbars hr table emoticons image',
+  statusbar: false,
+  toolbar:
+    'formatselect forecolor | alignleft aligncenter alignright alignjustify | bullist numlist indent outdent | link unlink | image table emoticons hr | code',
+  quickbars_selection_toolbar:
+    'bold italic underline strikethrough | h1 h2 h3 | quicklink blockquote removeformat',
+  quickbars_insert_toolbar: false,
+  width: '100%',
+};
+
+// Overrides some of the TinyMCE theme to better match the Admin UI style
+const GlobalStyles = () => (
+  <Global
+    styles={css`
+      .tox-tinymce {
+        border-radius: 5px !important;
+        border-color: #c1c7d0 !important;
+      }
+    `}
+  />
+);
+
+const WysiwygField = ({ onChange, autoFocus, field, errors, value: serverValue }) => {
+  const handleChange = value => {
+    if (typeof value === 'string') {
```

Comment:
Removed the check against `serverValue` since `ItemDetails` does that check now. It might be more performance-friendly to add an option to disable that check there, though, since these posts can get large.